ipapython/ipautil.py

```diff
@@ -142,6 +153,7 @@ def __init__(self, addr, match_local=False, parse_netmask=True,
 
         if isinstance(addr, CheckedIPAddress):
             self.prefixlen = addr.prefixlen
+            self._net = addr._net
```

Comment:
Yes, but in __init__ of CheckedIPAddress, there is custom initialization of self._net, so creating new instance using copying might get different values (None) in new object by using just super call without explicit variable assigment.